prow/cluster/jobs/istio/istio/istio.istio.master.yaml

```diff
@@ -105,6 +105,7 @@ presubmits:
       branches: *branch_spec
       agent: kubernetes
       context: prow/istio-pilot-multicluster-e2e.sh
+      optional: true
```

Comment:
Multicluster is important to my org.  I'd like a process to get this on the required test list / merge gate.

Reply:
@costinm - Isn't it accurate that promotions of e2e tests to required require TOC approval?  I think you mentioned this in relationship to coverage test cases which were moved to approved without TOC oversight.  In this case, optional should have been where this test started out - however, I think @tiswanso struggles to find the prow documentation so this PR could have started out optional (as it should have).

Reply:
We can change this to required only after the test has been proven to be stable for a week. Otherwise, we risk a flood of PRs stuck on this. Flipping the required bit is a small change that should not require PRs to rebase.

While you may have tested this a couple of times, we need to see how this holds up when we have 80+ PRs, and several of them doing /retest. prow e2e tests are barely holding up for single cluster tests, and take a very long time.

IF you want this test to gate a release, then please add it to the daily release tasks. Those run in the background and can have as many required tests as we wish.

Reply:
@rshriram -- Yep.. agree with the points.  Admittedly, I didn't expect the test-infra PR to merge so quickly after submitting so unfortunately this discussion didn't happen in that context.  My appologies for the noise.  This test has been pending merge for months and I've been triggering it manually basically multiple times a day throughout rel1.0 so it's not "new" in that sense, but yes, it's not been in the presubmit PR queue prior so the load hasn't been applied.

Thanks for clarifying the process.

Reply:
/lgtm

Reply:
Changes to test infrastructure and required tests should be broadly announced - since it affects all developers. And if developers raise issues - the TOC is the right forum to decide.